mailsync

```diff
@@ -1,5 +1,84 @@
 #!/bin/bash
-migsync etc/${1%/} ~/etc
-migsync etc/${1%/} ~/etc
-migsync mail/${1%/} ~/mail
-migsync mail/${1%/} ~/mail
\ No newline at end of file
+
+# check to see if there's an argument
+
+if [ -z "$1" ]; then
+  echo -e 'USAGE: mailsync [domain]'
+  exit 1
+fi 
+
+if [ ! -d ~/migration/migstuff/ ] ; then  
+  mkdir ~/migration/migstuff/ 2>/dev/null
+  mkdir ~/migration/migstuff/logs/ 2>/dev/null
+elif [ ! -d ~/migration/migstuff/logs/ ] ; then 
+  mkdir ~/migration/migstuff/logs/ 2>/dev/null
+  mkdir ~/migration/migstuff/logs/"$1"/ 2>/dev/null
+fi
+
+echo ''
+timestamp=$(date +%s)
+
+# function to count inodes in mail dir for domain 
+
+echo ' ' > ~/migration/migstuff/logs/"$1"/beforesync_"$1"_"$timestamp".txt
+echo ' ' > ~/migration/migstuff/logs/"$1"/aftersync_"$1"_"$timestamp".txt
+
+function mail_Count() {
+  (cd ~/mail/"$1" ; find . -printf "%h\n" | cut -d/ -f-2 | sort | uniq -c | sort -rn)
```

Comment:
```suggestion
  (find ~/mail/$1 -printf "%h\n" | cut -d/ -f-2 | sort | uniq -c | sort -rn)
```

Reply:
had to reverse this, not sure why it wasn't working. will work on improving this.